### DISCUSSION TITLE
[Future Request] Enhancing changes to make React Day Picker responsive

### DISCUSSION THREAD

**Discussion by rztma65020** [gpbl/react-day-picker#2163] [Ideas]
### Discussed in https://github.com/gpbl/react-day-picker/discussions/2067

<div type='discussions-op-text'>

<sup>Originally posted by **rztma65020** March  4, 2024</sup>
Hello team, 
Please update additional changes to make React Day Picker as a mobile friendly component. This would add a benefit in terms of being responsive. Here are some changes that we can add in the next release to make it responsive. I hope this will add value to the project. Thank you.

/* Mobile CSS updates*/

@media screen and (max-width: 650px) {
    .rdp-months {
        display: inline-block;
    }
    .rdp-month {
        margin: 0;
    }
    .rdp,.root_qxxl {
        --rdp-cell-size: 45px;
        --rdp-caption-font-size: 16px;
    }
}</div>

**Comment by jhoanborges** [comment]
this worked for me

```
@media screen and (max-width: 650px) {
    .rdp-root {
        --rdp-day-height: '2rem'
    }
}
```


![image](https://github.com/user-attachments/assets/cb396f56-8348-411b-b4ba-f4b9146ff0f3)
